k8s/geocoder/data-importers/csv-import/csv-importer.yaml

```diff
@@ -4,30 +4,43 @@ metadata:
   name: csv-importer
   namespace: geocoder
 spec:
-  backoffLimit: 0 
+  backoffLimit: 0
   template:
     metadata:
       annotations:
         sidecar.istio.io/inject: "false"
     spec:
+      initContainers:
+      - name: fetch-files
+        image: google/cloud-sdk:latest
+        command: ["/bin/sh", "-c"]
+        args:
+          - |
+            # Find the latest CSV file in the bucket
+            LATEST_FILE=$(gsutil ls gs://pelias-bucket | sort | tail -n 1)
+            # Copy the latest CSV file to the shared volume
+            gsutil cp ${LATEST_FILE} /data/csv-importer-files/
```

Comment:
gsutil doesn't see to create the target directory in case it is not already exists. I , therefore, added `mkdir -p /data/csv-importer-files/`  before copying the file to avoid this error.